scripts/test-sim-versions.yaml

```diff
@@ -0,0 +1,28 @@
+- stage1:
```

Comment:
I wonder if we can get jenkins to only run the test if this file has been updated. Maybe a question to ask Stefan.

Reply:
hmm don't we want the release that we're currently working on to be tested regularly against these older versions though?